circuits/cpp/src/aztec3/circuits/rollup/root/.test.cpp

```diff
@@ -183,7 +180,7 @@ TEST_F(root_rollup_tests, native_check_block_hashes_empty_blocks)
     EXPECT_FALSE(composer.failed());
 
     // Expected hash of public inputs for an empty L2 block. Also used in the contract tests.
-    fr const expected_hash = uint256_t("11840efc30e9fcbdd0aae30da2a5b441132420b4f0cc4ffd6bdc41888845f775");
+    fr const expected_hash = uint256_t("23f698e487c4e1b7383889f8864bc19591773d136c364c01d980bd13d0786ba1");
```

Comment:
How do we recalculate this value? Is it viable to calculate it on the fly in the test instead of hardcoding?

Reply:
Its values from the `l2-block-publisher.test.ts`, there to make sure that they are on par.